db/migrate/20160307161402_add_scheduled_reports.rb

```diff
@@ -0,0 +1,13 @@
+class AddScheduledReports < ActiveRecord::Migration
+  def change
+    create_table :scheduled_reports do |t|
+      t.string :name, null: false
+      t.string :frequency, null: false
```

Comment:
perhaps I am not understanding how AR enums work.

I expect that an enum is an array of possible integer values, that are mapped to string constants in the AR class. If I wanted to save an arbitrary crontab string like `0 1 3 * 2` how would I support that with an enum?